### PR TITLE
Add information about property descriptions

### PR DESCRIPTION
**Pull request by ekoleda-codaio**
To support the launch of sync table column descriptions.

### CODE REVIEW

File: docs/guides/advanced/schemas.md [hunk 1]
```diff
@@ -136,6 +136,70 @@ let MovieSchema = coda.makeObjectSchema({
...
+  },
+  // ...
+});
+```
+
+These descriptions are shown in the formula editor when inspecting a property.
+
+<img src="../../../images/schemas_descriptions.png" srcset="../../../images/schemas_descriptions_2x.png 2x" class="screenshot" alt="A schema property description being shown in the formula editor">
+
+If the schema is used in a sync table the descritpions will also be used to populate the initial [column descriptions](#column-descriptions).
```
Comment (jonathan-codaio) [inline docs/guides/advanced/schemas.md:169]: Nit, typo on descritpions

File: docs/guides/advanced/schemas.md [hunk 2]
```diff
@@ -136,6 +136,70 @@ let MovieSchema = coda.makeObjectSchema({
...
+
+??? tip "Add descriptions to object properties"
+
+    If a schema is only used by a single property, or you want to show the decription everywhere, then simply add the description to the schema.
```
Comment (jonathan-codaio) [inline docs/guides/advanced/schemas.md:173]: Nit, typo on decription

File: docs/guides/advanced/schemas.md (hunk 1 above)
Comment (jonathan-codaio) [inline docs/guides/advanced/schemas.md:169]: If the schema is used as a column format, when projecting child properties using the column header menu or value dropdown the column description will also be populated.

**Review by jonathan-codaio — APPROVED** [review]

File: docs/guides/advanced/schemas.md (hunk 2 above)
Comment (ekoleda-codaio) [reply docs/guides/advanced/schemas.md]: Ditto.

File: docs/guides/advanced/schemas.md (hunk 1 above)
Comment (ekoleda-codaio) [reply docs/guides/advanced/schemas.md]: Good callout. I've removed the column descriptions section and instead made a tabbed widget with the different places that property descriptions are displayed. PTAL.

**Review by jonathan-codaio — APPROVED** [review]

**Pull request merged**